database/seeds/FakePlayerSeeder.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Seeder;
+use App\Player;
+
+class FakePlayerSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        // Only run this if you're on a local environment
+        if(env('APP_ENV') === 'local') {
+            for ($x = 0; $x <= 3000; $x++) {
+                $p = new Player;
+                $p->email = str_random(12).'@gmail.com';
```

Comment:
Is there any risk of this leading to emails being sent out to random real email addresses during testing?

Reply:
I've changed it to `@fake.fake`